DICUI/MainWindow.xaml.cs

```diff
@@ -228,6 +235,8 @@ private void PopulateSystems()
 
             List<KnownSystemComboBoxItem> comboBoxItems = new List<KnownSystemComboBoxItem>();
 
+            comboBoxItems.Add(new KnownSystemComboBoxItem(KnownSystem.NONE));
```

Comment:
Check for an "Unknown" entry under the first group as well, it seems like it might have been shifted to there originally, hence I thought it was "missing".

Reply:
Yes it's not a real system so it shouldn't be returned from the list but added manually. This is fixed now.